capellambse_context_diagrams/serializers.py

```diff
@@ -91,6 +92,9 @@ def make_diagram(
         for edge, ref, parent in self._edges.values():
             self.deserialize_child(edge, ref, parent)
 
+        for junction, ref, parent in self._junctions.values():
+            self.deserialize_child(junction, ref, parent)
+
         self.diagram.calculate_viewport()
         self.order_children()
         self._edges.clear()
```

Comment:
```suggestion
```

This is no longer necessary now that serializers aren't being reused anymore.

Reply:
It is very needed. And I added `self._junctions.clear()` too.